src/main/kotlin/no/nav/syfo/kafka/consumers/varselbus/domain/EsyfovarselHendelse.kt

```diff
@@ -81,6 +81,7 @@ enum class HendelseType {
     SM_AKTIVITETSPLIKT,
     SM_ARBEIDSUFORHET_FORHANDSVARSEL,
     SM_VEDTAK_FRISKMELDING_TIL_ARBEIDSFORMIDLING,
+    SM_VARSEL_MANGLENDE_MEDVIRKNING,
```

Comment:
Hvis vi skal ha flere varslinger fra `ismanglendemedvirkning` bør de gå under denne hendelsen eller ønsker vi forskjellige hendelser basert på hvilken vurdering som første til hendelsen?

Reply:
Dersom det er helt ulike ting så kan det sikkert være greit med ulike typer. Om du ser på f.eks dialogmøte, så er der ulike hendelser for innkalling, referat osv.

Men - hva er forskjellen på det bruker opplever? Hva betyr "flere varslinger"? Dersom det er samme handling som skal skje uansett, som for eksempel distribuere journalpost, så gir det kanskje ikke så mye verdi å splitte? Eksempelvis for dialogmøte så er det ulikt innhold i varselet, og litt ulike ting som skjer for bruker. (visning av mikrofrontend osv)

Reply:
Om vi skal ha flere varslinger her så skal det nok være forskjellig innhold ja. Dette er forhåndsvarselet om stans.
Hva tenker egentlig dere på som et "varsel"? Er det en hvilken som helst notification til bruker?

Reply:
Jepp, det har dessverre blitt litt bloated.

Varsel for sykmeldt kan være:
- Varsel på min side via brukernotifikasjoner
- Varsel på ditt sykefravær
- Mikrofrontend på min side
- Varsel via dokumentdistribusjon (krever journalpost-id)

..eller en kombinasjon av disse om man ønsker flere. For deres tilfelle her er det vel bare dokumentdistribusjon som er aktuelt her :)

Reply:
Ønsker helst å endre DTO for esyfovarsel slik at den ikke har kjennskap til domene-ting (ala hendelsetype), men at den lar domene-appene selv velge hva som skal varsles hvor. Men det får bli til en annen gang!

Reply:
Så et varsel er en hvilken som helst beskjed som blir synliggjort for bruker?

Reply:
Det kan du sikkert si. Det blir synlig for bruker ja, krever ikke alltid en handling. For eksempel når vi sender varsel om dialogmøtereferat så blir det en brukernotifikasjon-beskjed, som bare opplyser på min side om at det finnes et referat (og lenker til det). Men ingen handling kreves

Reply:
Alright, takk for svar 🙌